Sources/FaceLiveness/Views/GetReadyPage/CameraPreviewView.swift

```diff
@@ -39,6 +39,8 @@ struct CameraPreviewView: View {
                     .position(x: geometry.size.width*Self.previewXPositionRatio,
                               y: geometry.size.height*Self.previewYPositionRatio)
             }
+        }.onDisappear {
+            model.stopSession()
```

Comment:
Just leaving a note here to confirm if we are not doing a long running task here (i assume we don't)

Reply:
correct, this isn't meant to be long running or UI-blocking